k4Interface/include/k4Interface/IGeoSvc.h

```diff
@@ -32,9 +32,10 @@ class G4VUserDetectorConstruction;
 class GAUDI_API IGeoSvc : virtual public IService {
 public:
   DeclareInterfaceID(IGeoSvc, 1, 0);
-  virtual dd4hep::DetElement           getDD4HepGeo() = 0;
-  virtual dd4hep::Detector*            lcdd()         = 0;
-  virtual G4VUserDetectorConstruction* getGeant4Geo() = 0;
+  virtual dd4hep::DetElement           getDD4HepGeo()                            = 0;
+  virtual dd4hep::Detector*            lcdd()                                    = 0;
```

Comment:
Not really part of this PR but on the bigger picture, should this be named `getDetector` or something slightly less specific than `lccd`?

Reply:
Done